libipl/p10/ipl0.C

```diff
@@ -682,7 +683,14 @@ static int ipl_updatehwmodel(void)
 	    (!fs::exists(BOOTTIME_GUARD_INDICATOR)))
 		apply_fco_override();
 
-	process_guard_records();
+	try {
```

Comment:
Can you please move the try and catch block inside the `process_guard_records` function?